src/sentry/tagstore/snuba/backend.py

```diff
@@ -743,6 +750,46 @@ def get_tag_value_paginator_for_projects(
                 ]
             )
 
+        if key == SEMVER_ALIAS:
+            # If doing a search on semver, we want to hit postgres to query the releases
+            version = query
+            if not version:
+                version = "*"
+            elif version[-1] not in SEMVER_WILDCARDS | {"@"}:
+                suffix = ""
+                if version[-1] != ".":
+                    suffix += "."
+                suffix += "*"
+                version += suffix
+            organization_id = Project.objects.filter(id=projects[0]).values_list(
+                "organization_id", flat=True
+            )[0]
+            if version[:-2] and "@" not in version and re.search(r"[^\d.]", version[:-2]):
+                # Handle searching just on package
```

Comment:
Starting to wonder if this all just belongs in the parser, then we could be more permissive with searches on the FE as well